paddle/fluid/imperative/layer.cc

```diff
@@ -394,5 +394,14 @@ std::shared_ptr<GradOpNode> CreateGradOpNode(
   }
 }
 
+void BumpInplaceVersion(std::shared_ptr<imperative::VarBase> var) {
```

Comment:
Can we make it as a member function of VarBase?

Reply:
Yes, done